Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,30 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+
+//task duration in .... minutes
+const taskDurationMinutes = tasks.map(function(task) {
+  return task.duration;
+})
+
+console.log(taskDurationMinutes); //[ 180, 120, 20, 200, 240, 180, 10, 200, 40 ]
+
+// task duration in ... hours
+const taskDuartionHours = taskDurationMinutes.map(function(workHours){
+  return workHours / 60;
+}) 
+console.log(taskDuartionHours);//[ 3, 2, 0.33, 3.33, 4, 3, 0.17, 3.33, 0.67]
+
+//filter the durations which are less than 2hr
+const earningHours = taskDuartionHours.filter(function(payHours) {
+    return payHours >= 2;
+})
+console.log(earningHours);//[ 3, 2, 3.33, 4, 3, 3.33]
+
+//total earning .. let earning per hour is 10
+const earning = earningHours
+    .map(hours => hours * 10)
+    .reduce ((sum, pay) => sum + pay);
+
+    console.log(earning);//186.66666666666669
+    console.log(earning.toFixed(2)); //186.67
```

Comment:
Nice work with formatting the output.  As a tip, have a look a the string method `toLocaleString`.  It is very powerful.